docs/guide/install-and-run.asciidoc

```diff
@@ -4,14 +4,14 @@
 To get started using Elastic APM,
 you need to have:
 
-* an Elasticsearch cluster and Kibana (version 5.6 or above)
-* APM Server
-* APM agents installed in your services
+* {apm-agents-ref}/index.html[APM agents] installed in your services
+* {apm-server-ref-v}/index.html[APM Server]
+* An {ref}/index.html[Elasticsearch] cluster (version 5.6+)
```

Comment:
v6.2 is the minimum supported version of Elasticsearch for use the Elastic APM

Reply:
well shoot, that's good to know